nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestDetectDuplicate.java

```diff
@@ -31,22 +25,40 @@
 import org.apache.nifi.logging.ComponentLog;
 import org.apache.nifi.reporting.InitializationException;
 import org.apache.nifi.state.MockStateManager;
-import org.apache.nifi.util.MockControllerServiceInitializationContext;
 import org.apache.nifi.util.MockComponentLog;
+import org.apache.nifi.util.MockControllerServiceInitializationContext;
 import org.apache.nifi.util.TestRunner;
 import org.apache.nifi.util.TestRunners;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 
 public class TestDetectDuplicate {
 
-    static {
+    @BeforeAll
+    public static void setup() {
         System.setProperty("org.slf4j.simpleLogger.defaultLogLevel", "info");
         System.setProperty("org.slf4j.simpleLogger.showDateTime", "true");
         System.setProperty("org.slf4j.simpleLogger.log.nifi.io.nio", "debug");
         System.setProperty("org.slf4j.simpleLogger.log.nifi.processors.standard.DetectDuplicate", "debug");
         System.setProperty("org.slf4j.simpleLogger.log.nifi.processors.standard.TestDetectDuplicate", "debug");
     }
 
+    @AfterAll
+    public static void cleanup() {
+        System.clearProperty("org.slf4j.simpleLogger.defaultLogLevel");
+        System.clearProperty("org.slf4j.simpleLogger.showDateTime");
+        System.clearProperty("org.slf4j.simpleLogger.log.nifi.io.nio");
+        System.clearProperty("org.slf4j.simpleLogger.log.nifi.processors.standard.DetectDuplicate");
+        System.clearProperty("org.slf4j.simpleLogger.log.nifi.processors.standard.TestDetectDuplicate");
+    }
```

Comment:
Additional logging should not be necessary in tests, so this is a good opportunity to remove all of these system properties.

Reply:
Removing them